src/routes/index.js

```diff
@@ -2,11 +2,13 @@ import express from 'express';
 import authRoute from './auth.route';
 import tripRoute from './trip.route';
 import profileSettingsRoute from './profile-settings.route';
+import accommodationRoute from './accommodation.route';
 
 const app = express();
 
 app.use('/api/auth', authRoute);
 app.use('/api/', tripRoute);
 app.use('/api/', profileSettingsRoute);
+app.use('/api/', accommodationRoute);
```

Comment:
The route context should be `/accommodations` instead

Reply:
Did you see this comment? @higustave-ops

Reply:
We're using this index file to collect all routes together. that /accommodations you're saying is in the accommodation route file.  let me know if I'm getting it wrong

Reply:
The essence of adding the prefix here is to avoid reputation.

Also, the route tree should look like this

- `POST /`
- `GET /types`.

Accessing it from a client, the route tree will look like this

- `POST /accommodations`
- `GET /accommodations/types`.